doc/nrf/app_dev/wifi_coex/index.rst

```diff
@@ -1,103 +1,113 @@
 .. _ug_radio_coex:
 
-Adding Wi-Fi Coexistence support to short-range radio applications
-##################################################################
+Coexistence of short-range radio and other radios
```

Comment:
Is this change based on some PMT requirements? Although surely technically correct, I wonder if there really are other use cases then using this with Wi-Fi. Even if there are I doubt we can really say they are supported, as until now we've only tested it with Wi-Fi. I'm afraid if for the sake of precision we aren't loosing clearness of the text for people using the most common use case.

However if we go with the generic "other radios" I would suggest adding "e.g. WiFi" next to each "other protocols" statement .

Reply:
Exactly what @jciupis said. There is Bluetooth-only 1-wire coexistence, which is for LTE. The term is intentionally wider than just Wi-Fi.

Reply:
The term Wi-Fi or LTE is in the page content. Search will match. By the way on sdk-nrf repo level this is now the only chapter  about any coexistence. Especially moved misplaced Bluetooth-only 1-wire into this chapter.